packages/cadl-ranch-specs/http/azure/core/traits/mockapi.ts

```diff
@@ -17,6 +17,14 @@ Scenarios.Azure_Core_Traits_smokeTest = passOnSuccess(
     if (req.params.id !== "1") {
       throw new ValidationError("Expected path param id=1", "1", req.params.id);
     }
+
+    if (!("repeatability-request-id" in req.headers)) {
```

Comment:
Are all headers in `req.headers` always lower case?

Reply:
Yes, I actually tested explicitly :)
![image](https://github.com/Azure/cadl-ranch/assets/1050156/14773736-616f-402c-9738-e9927a87b9e2)